src/sensitivities/array.jl

```diff
@@ -1,3 +1,11 @@
+import Base: size, length, reshape
+
 # Let the user get the `size` and `length` of `Node`s.
 Base.size(x::Node, dims...) = size(x.val, dims...)
 Base.length(x::Node) = length(x.val)
+
+# Sensitivity for the first argument of `reshape`.
+@explicit_intercepts reshape Tuple{∇RealArray, Vararg{Int64}} [true, false]
+@explicit_intercepts reshape Tuple{∇RealArray, Tuple{Vararg{Int64}}} [true, false]
```

Comment:
Sorry, just noticed you're using `Int64` here. You should use `Int`, as this will use whatever the system-appropriate representation is. If you type `Int` on the REPL, it will show `Int64` because we're both using 64-bit systems. It would be 32-bit ints on a 32-bit system.

Reply:
Aha, didn't know `Int` existed. Sweet. For some reason `@edit` is broken here:

```
julia> @edit
ERROR: MethodError: no method matching @edit()
Closest candidates are:
  @edit(::ANY) at interactiveutil.jl:413
```